### PR TITLE
feat: remove cloud and login on import

### PR DESCRIPTION
**Pull request by elboy3**
***What existing issue does this pull request close?***

Deprecate cloud from dq https://app.shortcut.com/galileo/story/10812/dq-deprecate-cloud-from-dq?vc_group_by=day&ct_workflow=all&cf_workflow=500000005

Do not require login on dq import https://app.shortcut.com/galileo/story/10813/dq-do-not-require-login-on-dq-import

***Demonstration***

After:

```
>>> import dataquality as dq
>>> dq.login()
Welcome to Galileo 1.5.0!
To skip this prompt in the future, set the following environment variable: GALILEO_CONSOLE_URL
🔭 Enter the url of your Galileo console
console.dev.rungalileo.io
📡 https://console.dev.rungalileo.io
🔭 Logging you into Galileo

Go to https://console.dev.rungalileo.io/get-token to generate a new API Key
🔐 Enter your API Key:
🚀 You're logged in to Galileo as galileo@rungalileo.io!

>>> exit()
(.venv) elliottchartock@Elliotts-MacBook-Pro dataquality % python
Python 3.9.6 (default, Feb 15 2022, 16:41:37) 

>>> import dataquality as dq
>>> dq.login()
✅ Already logged in as galileo@rungalileo.io!
Use logout() if you want to change users
```

Before:

```
>>> import dataquality as dq
Welcome to Galileo Cloud 1.4.2!
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/elliottchartock/Code/dataquality/dataquality/__init__.py", line 39, in <module>
    import dataquality.core._config
  File "/Users/elliottchartock/Code/dataquality/dataquality/core/__init__.py", line 6, in <module>
    from dataquality.analytics import Analytics
  File "/Users/elliottchartock/Code/dataquality/dataquality/analytics.py", line 10, in <module>
    from dataquality.clients.api import ApiClient
  File "/Users/elliottchartock/Code/dataquality/dataquality/clients/api.py", line 12, in <module>
    from dataquality.core._config import config
  File "/Users/elliottchartock/Code/dataquality/dataquality/core/_config.py", line 254, in <module>
    config = set_config()
  File "/Users/elliottchartock/Code/dataquality/dataquality/core/_config.py", line 240, in set_config
    set_platform_urls(console_url_str=console_url)
  File "/Users/elliottchartock/Code/dataquality/dataquality/core/_config.py", line 121, in set_platform_urls
    _validate_api_url(console_url_str, api_url)
  File "/Users/elliottchartock/Code/dataquality/dataquality/core/_config.py", line 140, in _validate_api_url
    raise GalileoException(err_detail.format(err=str(e))) from None
dataquality.exceptions.GalileoException: The provided console URL https://console.cloud.rungalileo.io is invalid or is not currently available. If you are sure this is correct, reach out to your admin.

To change your console url, run: 
`os.environ['GALILEO_CONSOLE_URL']='URL'` and then 
`dq.configure()`

Detail: HTTPSConnectionPool(host='api.cloud.rungalileo.io', port=443): Max retries exceeded with url: /healthcheck (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x10c3af0a0>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known'))
```

## Other info

Pin opencv version, 4.9.0.80 introduced typing issues  https://pypi.org/project/opencv-python/#history

### CODE REVIEW

**Review by setu4993 — APPROVED** [review]
Nice cleanup 🔥.

File: README.md [hunk 1]
```diff
@@ -24,9 +24,9 @@ Install the package.
 pip install dataquality
 ```
 
-Create an account at [Galileo](https://console.cloud.rungalileo.io/sign-up)
+Create an account at [Galileo](https://console-url.rungalileo.io/sign-up)
```
Comment (ShuaiShao93) [inline README.md:27]: What is console-url? Maybe better to use `{console-url} ` if it's an variable?

File: dataquality/__init__.py [hunk 2]
```diff
@@ -144,21 +131,6 @@
...
-#  Logging is disabled by default for enterprise users.
-#  To enable logging, set the environment variable
-#  DQ_TELEMETRICS=1
-#  To log initiate the Analytics class and pass in the gallileo ApiClient + dq.config
-#  a = Analytics(ApiClient, config)
-#  Once initialized you can start logging
-#  a.log_import("dataquality")
-#  a.log_method_call("dataquality.log_data_samples")
-a = Analytics(ApiClient, config)
-a.log_import("dataquality")
```
Comment (elboy3) [inline dataquality/__init__.py:159]: don't need any of this anymore

File: README.md [hunk 3]
```diff
@@ -50,7 +50,7 @@ By setting the token, you'll never be prompted to log in
 ```python
 import dataquality as dq
 
-dq.config.token = 'MY-TOKEN'
+config.token = 'MY-TOKEN'
```
Comment (ShuaiShao93) [inline README.md:53]: Need to initialize config?

**Review by ShuaiShao93 — APPROVED** [review]
Thanks!

File: README.md (hunk 3 above)
Comment (elboy3) [reply README.md]: fixed

File: README.md (hunk 1 above)
Comment (elboy3) [reply README.md]: done

**Comment by elboy3** [comment]
https://app.shortcut.com/galileo/story/10812/dq-deprecate-cloud-from-dq

**Comment by elboy3** [comment]
https://app.shortcut.com/galileo/story/10813/dq-do-not-require-login-on-dq-import

**Pull request merged**